README.md

```diff
@@ -63,7 +63,7 @@ container images and config changes are propagated to the cluster.
 |[Under Armour](https://www.underarmour.com)|[VSHN](https://vshn.ch)|[Walmart Chile](https://www.walmartchile.cl)
 |[Weave Cloud](https://cloud.weave.works)|[Workable](https://www.workable.com)|[Workarea](https://www.workarea.com)
 |[Working Group Two](https://wgtwo.com)|[Yusofleet](https://yusofleet.com)|[Yad2](https://yad2.co.il)
-|[Zaaksysteem](https://zaaksysteem.nl)||
+|[Zaaksysteem](https://zaaksysteem.nl)|[Consorcio Chile](https://www.consorcio.cl)|[Multicaja S.A](https://www.multicaja.cl)
```

Comment:
Can you please keep the list in alphabetical order?

Reply:
ready!

Reply:
I see no difference, `Consorcio` is still listed after `Zaaksysteem` while it should be listed after `CLVR Cloud Ltd`.